### ISSUE TITLE
Update url for go getting goreturns

### ISSUE THREAD

**Issue by avjinder** [Microsoft/vscode-go#1546]
[goreturns](sourcegraph.com/sqs/goreturns) has been 404'd. It should be removed as the default formatter and from the wiki as well. gofmt or goimports are good candidates for replacing it.

RIP goreturns.

**Comment by ramya-rao-a** [comment]
The repo still exists: https://github.com/sqs/goreturns
https://sourcegraph.com/sqs/goreturns is what 404s

cc @sqs

**Comment by avjinder** [comment]
Praise the lord 🙏, goreturns lives.
The [link](https://github.com/Microsoft/vscode-go/wiki/Go-tools-that-the-Go-extension-depends-on) on the tools page points to the dead url. After installing this package, vscode kept failing to get goreturns, so I ended up using goimports.

Does the package use the dead url to get the dependency?

**Comment by ramya-rao-a** [comment]
We have been using then link https://sourcegraph.com/sqs/goreturns from the start. Let's wait till what @sqs has to say and then update the link if needed

**Comment by sqs** [comment]
Sorry for the confusion here!

I just updated the README to use https://github.com/sqs/goreturns as the main web URL and `github.com/sqs/goreturns` as the main import path.

(More info, if curious: The import path `sourcegraph.com/sqs/goreturns` still worked (`curl 'https://sourcegraph.com/sqs/goreturns?go-get=1'`). The web URL https://sourcegraph.com/sqs/goreturns does not, though. We special-cased a few URLs like this on sourcegraph.com (most URLs are like https://sourcegraph.com/github.com/sqs/goreturns), and that regressed. Will fix in 24-48 hours. Even with this fix, we had seen some instances where our CDN was interfering with `go get`s from AWS, so in the interest of time I decided to just use `github.com/` as the canonical URL and import path.)
